### ISSUE TITLE
Can not access website after remove it out block list.

### ISSUE THREAD

**Issue by vdhong** [TechnikEmpire/CitadelCore.Windows#8]
Hello,

I made an application read domains from file and block it, when file change the app read file again and block only domains in file. Example I have file domain.txt below:
```
phimmoi.net
vnexpress.net
```
First application work ok (it blocked domain phimmoi.net and vnexpress.net), then I delete phimmoi.net domain from file and hope can access phimmoi.net now but when I access phimmoi.net from browser it show below error in console and browser can not access phimmoi.net. I guest this proxy cache blocking this domain. Please help to check this issue.
```
ERRO: An error occurred while sending the request.   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Net.Http.HttpClient.<FinishSendAsyncUnbuffered>d__59.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at CitadelCore.Net.Handlers.FilterHttpResponseHandler.<Handle>d__6.MoveNext()The remote name could not be resolved: 'www.phimmoi.net'   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
         From FilterHttpResponseHandler.cs::Handle() #263
```
Note: I can access phimmoi.net when I turn off the application and then re-open it with after edit domains.txt

Thank you!

**Comment by TechnikEmpire** [comment]
Hey. Again I would need to see the relevant code. This filter is used in commercial software that constantly loads, flushes, updates and reloads complex filtering rules so I'm guessing that there's something else going on.

Plus, the attached error trace is showing something altogether different: a system DNS failure. Are you running some custom dns resolver? Perhaps this is related to your other bug ticket. See the remarks I made there, try changing things as I suggested and see if this issue also gets resolved.
